tests/tck/features/match/ZeroStep.feature

```diff
@@ -454,3 +454,39 @@ Feature: Variable length Pattern match (0 step)
       | []                                                             | ("Trail Blazers" :team{name: "Trail Blazers"})                                                              |
       | [[:like "Tony Parker"->"LaMarcus Aldridge" @0 {likeness: 90}]] | ("Trail Blazers" :team{name: "Trail Blazers"})                                                              |
       | []                                                             | ("Trail Blazers" :team{name: "Trail Blazers"})                                                              |
+
+  Scenario: Multiple variable length with edge filter
```

Comment:
This case is ok even without this modification.

Reply:
I forgot to remove the Maxhop `100`.